config/global.ini.php

```diff
@@ -49,6 +49,13 @@
 ;
 ;ignore_error_codes[] = 1105
 
+; Add a query hint for the order of joined tables when building segment queries in MySQL. This can be used to override
+; sub-optimal choices by the MYSQL optimizer and always ensure the query plan starts with the first table in the query.
+enable_segment_first_table_join_prefix = 0
+
+; Add a query hint for the order of the first table for all log table queries in MySQL.
+enable_first_table_join_prefix = 0
```

Comment:
```suggestion
; Add a query hint for the order of joined tables when building segment queries in MySQL. This can be used to override
; sub-optimal choices by the MySQL optimizer and always ensure the query plan starts with the first table in the query.
enable_segment_first_table_join_prefix = 0

; Add a query hint for the order of joined tables for all log table queries in MySQL.
enable_first_table_join_prefix = 0
```